dia/diameter_travelping.dia

```diff
@@ -0,0 +1,20 @@
+;; Copyright 2021, Travelping GmbH <info@travelping.com>
+
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License
+;; as published by the Free Software Foundation; either version
+;; 2 of the License, or (at your option) any later version.
+
+;; It contains custom Travelping AVPs related to NAT-binding
+
+@id     1
+@name   diameter_travelping
+@prefix diameter_travelping
+@vendor 18681 Travelping
+
+@avp_types
+
+   TP-NAT-IP-Address             16    OctetString     V
```

Comment:
please align the line from left to right, not from right to left.

Reply:
I can do it, but what is the difference? I mean if you will take a look at indentation of other dictionaries most of them done like this

Reply:
Based on
https://github.com/travelping/ergw_aaa/blob/a5264acef9d1ee265cba81f78d57cc055ff229a1/dia/diameter_3gpp_ts29_061_sgi_base_acc.dia
and https://github.com/travelping/ergw_aaa/blob/a5264acef9d1ee265cba81f78d57cc055ff229a1/dia/diameter_3gpp_ts29_061_sgi.dia
I see used one style - align the line from left to right

Reply:
It's only the data type that has different alignments. In the 3GPP dictionaries that column is right aligned, while it is left aligned in the RFC dictionaries.
I don't mind either style, as long as it is consistent within a file.

Reply:
How indentation of @avp_types related to the files you've pasted?

Reply:
The examples of files was provided based on:
>  I mean if you will take a look at indentation of other dictionaries most of them done like this
So, as you see in dictionaries what I provided above - alignment the line from left to right.